### PR TITLE
Project template for WPF

### PR DESCRIPTION
**Pull request by vatsan-madhavan**
This is the Project Template for WPF used by `dotnet new wpf` and _Create New Project_ -> _WPF App (.NET Core)_ in Visual Studio. 

Moving it from our internal repo build and ship from GitHub. We will make modifications to this in a future PR (likely to get this to match the template for .NET Framework).  

/cc @chabiss, @grubioe, @zsd4yr, @ericstj

### CODE REVIEW

**Comment by vatsan-madhavan** [comment]
Marking as *NO MERGE* just yet. The corresponding PR in the internal repo that _removes_ this Project template from that location hasn't been merged yet. We should complete that first before merging it here.

File: packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/MainWindow.xaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿<Window x:Class="Company.WpfApplication1.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:Company.WpfApplication1"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="450" Width="800">
```
Comment (stevenbrix) [inline packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/MainWindow.xaml]: did we want to change this to match what the default template is in net48?

Comment (vatsan-madhavan) [reply packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/MainWindow.xaml]: In a near-future change we'll probably want to do something like that. I think we'd want to spec what our project template ought to be like in consultation with @grubioe and @chabiss (Visual Studio) first.

**Comment by stevenbrix** [comment]
Will this fix #448

File: packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/MainWindow.xaml (hunk 1 above)
Comment (chabiss) [reply packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/MainWindow.xaml]: @vatsan-madhavan we need an item tracking this ASAP.  I would like the template to be changed for preview 5.

Comment (zsd4yr) [reply packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/MainWindow.xaml]: @chabiss, if you want to change the template in winforms, we should discuss as well on this thread dotnet/winforms#663

**Comment by vatsan-madhavan** [comment]
> Will this fix #448?

Doubt it. This is not a _change_ - this is just a _rehoming_.

**Pull request merged**